src/context.rs

```diff
@@ -226,6 +229,14 @@ impl<'gcc, 'tcx> CodegenCx<'gcc, 'tcx> {
         }
 
         let mut cx = Self {
+            int128_align: Align::from_bytes(
+                tcx.layout_of(ty::TypingEnv::fully_monomorphized().as_query_input(tcx.types.i128))
+                    .unwrap()
```

Comment:
Could you please avoid `unwrap()` in general and use `expect()` to provide a description?
Something like:
```suggestion
                    .expect("could not get the layout of i128")
```